pypeit/calibrations.py

```diff
@@ -312,6 +313,62 @@ def get_arc(self):
         self._update_cache('arc', 'arc', self.msarc)
         return self.msarc
 
+
+    def get_tiltimg(self):
+        """
+        Load or generate the Tilt image
+
+        Requirements:
+          master_key, det, par
+
+        Args:
+
+        Returns:
+            ndarray: :attr:`mstilt` image
+
+        """
+        # Check internals
+        self._chk_set(['det', 'calib_ID', 'par'])
+
+        # Prep
+        tilt_rows = self.fitstbl.find_frames('tilt', calib_ID=self.calib_ID, index=True)
+        if len(tilt_rows) == 0:
+            msgs.error('Must identify tilt frames to construct tilt image.')
```

Comment:
I've changed all the PypeIt files in the dev suite. Tilt is now a required frame type for a reduction. I think it is best to keep it explicit, which will reduce user error.

Reply:
Whenever the `tilt` frame type was introduced, I edited all the spectrograph `check_frame_type` methods to automatically type things as both `arc` and `tilt`.  I don't know if anyone has since changed how this works for each spectrograph.  So as long as people continue to use `pypeit_setup`, they shouldn't see any change.  But it may cause issues for people that try to use pypeit files from before the tilt frame type existed, which is what Joe had to do for the dev suite.

Reply:
Yeah, those old pypeit files are now going to be defunct unless people add tilt. We need to make sure setup actually types the arcs as tilt frames though.